lua/lvim/core/cmp.lua

```diff
@@ -1,3 +1,9 @@
+-- cmp's config requires cmp to be installed to run the first time
+local cmp_loaded, _ = pcall(require, "cmp")
+if not cmp_loaded then
+  return
+end
+
```

Comment:
This is redundant to https://github.com/LunarVim/LunarVim/blob/13f5cf436f279d882609a808654c1c5fe802d1ad/lua/lvim/core/cmp.lua#L155-L158

What errors are you encountering?